spec/models/tag_spec.rb

```diff
@@ -46,6 +46,16 @@
         tag.name = nil
         expect(tag).not_to be_valid
       end
+
+      it "fails validations if name uses diacritics" do
+        tag.name = "łookatmé"
+        expect(tag).not_to be_valid
+      end
+
+      it "fails validations if name uses non-latin characters" do
+        tag.name = "火"
```

Comment:
We could certainly expand this to include more character sets, but I think that might be overdoing it. :stuck_out_tongue:

Reply:
I don't think so tbh. I left a comment on the issue, but given our coming I18n efforts it seems restricting tags to ASCII is a bit counterproductive.

Reply:
@citizen428 I agree in the long run.

Reply:
Lol re-reading this comment, I meant adding more tests was overdoing it... :sweat_smile: 

I think we should work toward removing this limitation on tags, especially considering that in the future there might be non-English communities on Forem.